docs/CMakeLists.txt

```diff
@@ -66,6 +68,14 @@ add_custom_target(
   WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
   COMMENT "Build full documentation")
 
+add_custom_target(
+  docs-clean
+  COMMAND find ${sphinx_api} -mindepth 1 -type f -and -not -name "api.rst" -delete
+  COMMAND find ${sphinx_api} -mindepth 1 -type d -delete
+  COMMAND rm -rf ${sphinx_build}
+  COMMENT "Cleaning documentation artifacts"
+)
+
```

Comment:
I think these changes sneaked in but should disappear after merging https://github.com/acts-project/acts/pull/1393

Reply:
Should be gone.